src/components/KuadrantRateLimitPolicyCreatePage.tsx

```diff
@@ -203,7 +203,7 @@ const KuadrantRateLimitPolicyCreatePage: React.FC = () => {
                   </HelperText>
                 </FormHelperText>
               </FormGroup>
-              <NamespaceSelect selectedNamespace={selectedNamespace} onChange={setSelectedNamespace} />
+              <NamespaceSelect selectedNamespace={selectedNamespace} onChange={setSelectedNamespace} formDisabled={false} />
```

Comment:
this will be updated when update is in the other policies its just temp set to false